docs/config.md

```diff
@@ -52,7 +52,6 @@ This is a list of available options:
 | `askformore`     | `bool`   | If enabled - it will ask to add more data after use of `generate` command.  DEPRECATED in v1.10.0 |
 | `autoclip`       | `bool`   | Always copy the password created by `gopass generate`. Only applies to generate. |
 | `autoimport`     | `bool`   | Import missing keys stored in the pass repository without asking. |
-| `autosync`       | `bool`   | Always do a `git push` after a commit to the store. Makes sure your local changes are always available on your git remote. DEPRECATED in v1.10.0 |
```

Comment:
This says "DEPRECATED" already, and we keep deprecated config items in case for now.

Maybe we should move them to their own table, though?